x-pack/plugins/ml/public/application/management/jobs_list/components/access_denied_page.tsx

```diff
@@ -5,59 +5,46 @@
  * 2.0.
  */
 
-import React, { Fragment } from 'react';
+import React from 'react';
 
 import { FormattedMessage } from '@kbn/i18n/react';
-import { i18n } from '@kbn/i18n';
 
-import {
-  EuiCallOut,
-  EuiPage,
-  EuiPageBody,
-  EuiPageContentBody,
-  EuiPageContentHeader,
-  EuiPageContentHeaderSection,
-  EuiSpacer,
-  EuiText,
-  EuiTitle,
-} from '@elastic/eui';
+import { EuiEmptyPrompt, EuiPageContent, EuiPageHeader, EuiSpacer } from '@elastic/eui';
 
 export const AccessDeniedPage = () => (
-  <Fragment>
-    <EuiPage data-test-subj="mlPageAccessDenied">
-      <EuiPageBody>
-        <EuiPageContentHeader>
-          <EuiPageContentHeaderSection>
-            <EuiTitle>
-              <h1>
-                <FormattedMessage
-                  id="xpack.ml.management.jobsList.insufficientLicenseTitle"
-                  defaultMessage="Machine Learning"
-                />
-              </h1>
-            </EuiTitle>
-          </EuiPageContentHeaderSection>
-        </EuiPageContentHeader>
-        <EuiPageContentBody>
-          <EuiSpacer size="m" />
-          <EuiCallOut
-            title={i18n.translate('xpack.ml.management.jobsList.noPermissionToAccessLabel', {
-              defaultMessage: 'Access denied',
-            })}
-            color="danger"
-            iconType="cross"
-          >
-            <EuiText size="s">
-              <p>
-                <FormattedMessage
-                  id="xpack.ml.management.jobsList.noGrantedPrivilegesDescription"
-                  defaultMessage="You don’t have permission to manage ML jobs"
-                />
-              </p>
-            </EuiText>
-          </EuiCallOut>
-        </EuiPageContentBody>
-      </EuiPageBody>
-    </EuiPage>
-  </Fragment>
+  <>
+    <EuiPageHeader
+      data-test-subj="mlPageAccessDenied"
+      pageTitle={
+        <FormattedMessage
+          id="xpack.ml.management.jobsList.noPermissionToAccessPageTitle"
+          defaultMessage="Machine Learning"
+        />
+      }
+      bottomBorder
+    />
+
+    <EuiSpacer size="l" />
+    <EuiPageContent verticalPosition="center" horizontalPosition="center" color="danger">
+      <EuiEmptyPrompt
+        iconType="alert"
+        title={
+          <h2>
+            <FormattedMessage
+              id="xpack.ml.management.jobsList.noPermissionToAccessLabel"
+              defaultMessage="Access denied"
+            />
+          </h2>
+        }
+        body={
+          <p>
+            <FormattedMessage
+              id="xpack.ml.management.jobsList.noGrantedPrivilegesDescription"
+              defaultMessage="You don’t have permission to manage Machine Learning jobs."
```

Comment:
> defaultMessage="You don’t have permission to manage Machine Learning jobs.

Likewise, if there's a relatively short list of things they have to do/have to get past this error, that'd be nice to add here too.

Reply:
Added a section about the ML feature needing to be visible in the space in https://github.com/elastic/kibana/pull/103117/commits/8caf4d90e9754cf4c83e834f9a799642bbd77c50.